service/telemetry/internal/migration/v0.3.0.go

```diff
@@ -15,8 +15,10 @@ import (
 
 type TracesConfigV030 struct {
 	// Level configures whether spans are emitted or not, the possible values are:
-	//  - "none" indicates that no tracing data should be collected;
-	//  - "basic" is the recommended and covers the basics of the service telemetry.
+	//  - "None" indicates that no tracing data should be collected.
+	//  - "Basic" is the recommended and covers the basics of the service telemetry.
+	//  - "Normal" adds some other indicators on top of basic.
```

Comment:
Basic and Normal aren't documented here because their behaviour is no different from basic for traces.

Reply:
Is there anything currently enforcing the fact that `Basic`, `Normal`, and `Detailed` are all the same at this point?

Out of ignorance, my assumption is that at any point a change could be introduced that makes them different, so we may as well include valid values. Maybe we can add a note that they're currently the same thing? I just lean towards documenting valid values.